examples/active-class-name/components/Link.js

```diff
@@ -5,9 +5,10 @@ import React, { Children } from 'react'
 const ActiveLink = ({ router, children, ...props }) => {
   const child = Children.only(children)
 
-  let className = child.props.className || ''
+  let className = child.props.className || null
   if (router.pathname === props.href && props.activeClassName) {
-    className = `${className} ${props.activeClassName}`.trim()
+    className = `${className !== null ? className : ''} ${props.activeClassName}`.trim()
+    console.log(className)
```

Comment:
oops

Reply:
What idiot put that there?

 :nervous_grin: